### ISSUE TITLE
All ajax requests are failing when toolbar is shown

### ISSUE THREAD

**Issue by PapsOu** [symfony/symfony#26412]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | /no
| RFC?             | no
| Symfony version  | 3.4.5

The toolbar seems to break page ajax requests when it is shown. Ajax request are made with jQuery.get() function as regular usage.

I've tryed hiding the toolbar with following configuration :
```yaml
#app/config/config_dev.yml
web_profiler:
    toolbar: false
```
All ajax requests works fine.

Also, by setting the `excluded_ajax_paths` to '.*, it allows all requests to finish correctly.

```yaml
#app/config/config_dev.yml
web_profiler:
    toolbar: true
    excluded_ajax_paths: '.*'
```
It also affects a SF4 basic project without any vendors other than symfony bundles and components.

Here the composer info :
```
behat/gherkin                                   v4.5.1  Gherkin DSL parser for PHP 5.3
behat/transliterator                            v1.2.0  String transliterator
bentools/cartesian-product                      1.2     A simple, low-memory footprint function to generate all combinations from a multi-dimensionnal array.
clue/stream-filter                              v1.4.0  A simple and modern approach to stream filtering in PHP
cocur/slugify                                   v3.1    Converts a string into a slug.
codeception/codeception                         2.4.0   BDD-style testing framework
codeception/phpunit-wrapper                     6.0.5   PHPUnit classes used by Codeception
codeception/stub                                1.0.2   Flexible Stub wrapper for PHPUnit's Mock Builder
composer/ca-bundle                              1.1.0   Lets you find a path to the system CA bundle, and includes a fallback to the Mozilla CA bundle.
data-uri/data-uri                               0.2.5   PHP DataURI component
data-uri/twig-extension                         1.0.1   Twig extension for data URI ; Converts data to the data URI Url scheme (RFC 2397)
doctrine/annotations                            v1.6.0  Docblock Annotations Parser
doctrine/cache                                  v1.7.1  Caching library offering an object-oriented API for many cache backends
doctrine/collections                            v1.5.0  Collections Abstraction library
doctrine/common                                 v2.8.1  Common Library for Doctrine projects
doctrine/data-fixtures                          v1.3.0  Data Fixtures for all Doctrine Object Managers
doctrine/dbal                                   v2.6.3  Database Abstraction Layer
doctrine/doctrine-bundle                        1.8.1   Symfony DoctrineBundle
doctrine/doctrine-cache-bundle                  1.3.2   Symfony Bundle for Doctrine Cache
doctrine/doctrine-fixtures-bundle               v2.4.1  Symfony DoctrineFixturesBundle
doctrine/doctrine-migrations-bundle             v1.3.1  Symfony DoctrineMigrationsBundle
doctrine/inflector                              v1.3.0  Common String Manipulations with regard to casing and singular/plural rules.
doctrine/instantiator                           1.1.0   A small, lightweight utility to instantiate objects in PHP without invoking their constructors
doctrine/lexer                                  v1.0.1  Base library for a lexer that can be used in Top-Down, Recursive Descent Parsers.
doctrine/migrations                             v1.6.2  Database Schema migrations using Doctrine DBAL
doctrine/orm                                    v2.5.14 Object-Relational-Mapper for PHP
egulias/email-validator                         2.1.3   A library for validating emails against several RFCs
elasticsearch/elasticsearch                     v6.0.1  PHP Client for Elasticsearch
facebook/webdriver                              1.5.0   A PHP client for Selenium WebDriver
fig/link-util                                   1.0.0   Common utility implementations for HTTP links
friendsofsymfony/elastica-bundle                v5.0.3  Elasticsearch PHP integration for your Symfony project using Elastica
friendsofsymfony/jsrouting-bundle               1.6.3   A pretty nice way to expose your Symfony2 routing to client applications.
friendsofsymfony/oauth-server-bundle            1.5.2   Symfony2 OAuth Server Bundle
friendsofsymfony/oauth2-php                     1.2.3   OAuth2 library
friendsofsymfony/rest-bundle                    2.3.1   This Bundle provides various tools to rapidly develop RESTful API's with Symfony
fzaninotto/faker                                v1.7.1  Faker is a PHP library that generates fake data for you.
gedmo/doctrine-extensions                       v2.4.33 Doctrine2 behavioral extensions
guzzlehttp/guzzle                               6.3.0   Guzzle is a PHP HTTP client library
guzzlehttp/promises                             v1.3.1  Guzzle promises library
guzzlehttp/psr7                                 1.4.2   PSR-7 message implementation that also provides common utility methods
guzzlehttp/ringphp                              1.1.0   Provides a simple API and specification that abstracts away the details of HTTP into a single PHP function.
guzzlehttp/streams                              3.0.0   Provides a simple abstraction over streams of data
h4cc/wkhtmltoimage-amd64                        0.12.4  Convert html to image using webkit (qtwebkit). Static linked linux binary for amd64 systems.
h4cc/wkhtmltopdf-amd64                          0.12.4  Convert html to pdf using webkit (qtwebkit). Static linked linux binary for amd64 systems.
hamcrest/hamcrest-php                           v2.0.0  This is the PHP port of Hamcrest Matchers
html2text/html2text                             4.1.0   Converts HTML to formatted plain text
imagine/imagine                                 v0.7.1  Image processing for PHP 5.3
incenteev/composer-parameter-handler            v2.1.3  Composer script handling your ignored parameter file
jdorn/sql-formatter                             v1.2.17 a PHP SQL highlighting library
jeroendesloovere/vcard                          1.5.2   This VCard PHP class can generate a vCard with some data. When using an iOS device it will export as a .ics file because iOS devices don't support the default .vcf files.
jeroendesloovere/vcard-bundle                   1.2.2   This Symfony 2 bundle can generate vCards with lots of data. When using an < iOS 7 device it will export as a .ics file because iOS devices don't support the default .vcf files.
jms/metadata                                    1.6.0   Class/method/property metadata management in PHP
jms/parser-lib                                  1.0.0   A library for easily creating recursive-descent parsers.
jms/serializer                                  1.11.0  Library for (de-)serializing data of any complexity; supports XML, JSON, and YAML.
jms/serializer-bundle                           2.3.1   Allows you to easily serialize, and deserialize data of any complexity
knplabs/gaufrette                               v0.5.0  PHP library that provides a filesystem abstraction layer
knplabs/knp-components                          1.3.5   Knplabs component library
knplabs/knp-gaufrette-bundle                    v0.5.2  Allows to easily use the Gaufrette library in a Symfony project
knplabs/knp-menu                                2.3.0   An object oriented menu library
knplabs/knp-menu-bundle                         v2.2.1  This bundle provides an integration of the KnpMenu library
knplabs/knp-paginator-bundle                    v2.7.2  Paginator bundle for Symfony to automate pagination and simplify sorting and other features
knplabs/knp-snappy                              v1.0.4  PHP5 library allowing thumbnail, snapshot or PDF generation from a url or a html page. Wrapper for wkhtmltopdf/wkhtmltoimage.
knplabs/knp-snappy-bundle                       v1.5.2  Easily create PDF and images in Symfony2 by converting Twig/HTML templates.
kriswallsmith/assetic                           v1.4.0  Asset Management for PHP
leafo/scssphp                                   v0.7.5  scssphp is a compiler for SCSS written in PHP.
league/uri                                      5.2.0   URI manipulation library
league/uri-components                           1.7.1   URI components manipulation library
league/uri-hostname-parser                      1.1.1   ICANN base hostname parsing implemented in PHP.
league/uri-interfaces                           1.0.0   Common interface for URI representation
league/uri-manipulations                        1.4.0   URI manipulation library
league/uri-parser                               1.3.0   userland URI parser RFC 3986 compliant
league/uri-schemes                              1.1.1   URI manipulation library
lexik/maintenance-bundle                        v2.1.5  This bundle allows you to place your website in maintenance mode by calling two commands from your console.
liip/imagine-bundle                             1.9.1   This bundle provides an image manipulation abstraction toolkit for Symfony-based projects.
michelf/php-markdown                            1.8.0   PHP Markdown
mikey179/vfsStream                              v1.6.5  Virtual file system to mock the real file system in unit tests.
mockery/mockery                                 1.0     Mockery is a simple yet flexible PHP mock object framework for use in unit testing with PHPUnit, PHPSpec or any other testing framework. Its core goal is to offer a test double framework with a succinct API capable of clearly defining all possible object ope...
monolog/monolog                                 1.23.0  Sends your logs to files, sockets, inboxes, databases and various web services
myclabs/deep-copy                               1.7.0   Create deep copies (clones) of your objects
nelmio/alice                                    v2.3.4  Expressive fixtures generator
nelmio/api-doc-bundle                           2.13.3  Generates documentation for your REST API from annotations
ocramius/package-versions                       1.3.0   Composer plugin that provides efficient querying for installed package versions (no runtime IO)
ocramius/proxy-manager                          2.1.1   A library providing utilities to generate, instantiate and generally operate with Object Proxies
oro/doctrine-extensions                         1.2.0   Doctrine Extensions for MySQL and PostgreSQL.
pagerfanta/pagerfanta                           v1.0.5  Pagination for PHP 5.3
paragonie/random_compat                         v2.0.11 PHP 5.x polyfill for random_bytes() and random_int() from PHP 7
payum/iso4217                                   1.0.1   ISO 4217 PHP Library
payum/payum                                     1.5.0   One million downloads of Payum already! Payum offers everything you need to work with payments. Friendly for all PHP frameworks (Symfony, Laravel, Zend, Yii, Silex). Check more visiting site.
payum/payum-bundle                              2.3.0   One million downloads of Payum already! Payum offers everything you need to work with payments. Check more visiting site.
phar-io/manifest                                1.0.1   Component for reading phar.io manifest information from a PHP Archive (PHAR)
phar-io/version                                 1.0.1   Library for handling version information and constraints
php-http/guzzle6-adapter                        v1.1.1  Guzzle 6 HTTP Adapter
php-http/httplug                                v1.1.0  HTTPlug, the HTTP client abstraction for PHP
php-http/message                                1.6.0   HTTP Message related tools
php-http/message-factory                        v1.0.2  Factory interfaces for PSR-7 HTTP Message
php-http/promise                                v1.0.0  Promise used for asynchronous HTTP requests
phpcollection/phpcollection                     0.5.0   General-Purpose Collection Library for PHP
phpdocumentor/reflection-common                 1.0.1   Common reflection classes used by phpdocumentor to reflect the code structure
phpdocumentor/reflection-docblock               4.3.0   With this component, a library can provide support for annotations via DocBlocks or otherwise retrieve information that is embedded in a DocBlock.
phpdocumentor/type-resolver                     0.4.0  
phpoption/phpoption                             1.5.0   Option Type for PHP
phpspec/prophecy                                1.7.5   Highly opinionated mocking framework for PHP 5.3+
phpunit/php-code-coverage                       5.3.0   Library that provides collection, processing, and rendering functionality for PHP code coverage information.
phpunit/php-file-iterator                       1.4.5   FilterIterator implementation that filters files based on a list of suffixes.
phpunit/php-text-template                       1.2.1   Simple template engine.
phpunit/php-timer                               1.0.9   Utility class for timing
phpunit/php-token-stream                        2.0.2   Wrapper around PHP's tokenizer extension.
phpunit/phpunit                                 6.5.7   The PHP Unit Testing framework.
phpunit/phpunit-mock-objects                    5.0.6   Mock Object library for PHPUnit
polishsymfonycommunity/symfony-mocker-container v1.0.2  Provides base Symfony dependency injection container enabling service mocking.
psr/cache                                       1.0.1   Common interface for caching libraries
psr/container                                   1.0.0   Common Container Interface (PHP FIG PSR-11)
psr/http-message                                1.0.1   Common interface for HTTP messages
psr/link                                        1.0.0   Common interfaces for HTTP links
psr/log                                         1.0.2   Common interface for logging libraries
psr/simple-cache                                1.0.1   Common interfaces for simple caching
ramsey/uuid                                     3.7.3   Formerly rhumsaa/uuid. A PHP 5.4+ library for generating RFC 4122 version 1, 3, 4, and 5 universally unique identifiers (UUID).
react/promise                                   v2.5.1  A lightweight implementation of CommonJS Promises/A for PHP
ruflin/elastica                                 6.0.1   Elasticsearch Client
sebastian/code-unit-reverse-lookup              1.0.1   Looks up which function or method a line of code belongs to
sebastian/comparator                            2.1.3   Provides the functionality to compare PHP values for equality
sebastian/diff                                  2.0.1   Diff implementation
sebastian/environment                           3.1.0   Provides functionality to handle HHVM/PHP environments
sebastian/exporter                              3.1.0   Provides the functionality to export PHP variables for visualization
sebastian/global-state                          2.0.0   Snapshotting of global state
sebastian/object-enumerator                     3.0.3   Traverses array structures and object graphs to enumerate all referenced objects
sebastian/object-reflector                      1.1.1   Allows reflection of object attributes, including inherited and non-public ones
sebastian/recursion-context                     3.0.0   Provides functionality to recursively process PHP variables
sebastian/resource-operations                   1.0.0   Provides a list of PHP built-in functions that operate on resources
sebastian/version                               2.0.1   Library that helps with managing the version number of Git-hosted PHP projects
sensio/distribution-bundle                      v5.0.21 Base bundle for Symfony Distributions
sensio/framework-extra-bundle                   v3.0.29 This bundle provides a way to configure your controllers with annotations
sensio/generator-bundle                         v3.1.7  This bundle generates code for you
sensiolabs/security-checker                     v4.1.8  A security checker for your composer.lock
sonata-project/admin-bundle                     3.32.0  The missing Symfony Admin Generator
sonata-project/block-bundle                     3.12.0  Symfony SonataBlockBundle
sonata-project/cache                            2.0.1   Cache library
sonata-project/core-bundle                      3.9.1   Symfony SonataCoreBundle
sonata-project/datagrid-bundle                  2.3.1   Symfony SonataDatagridBundle
sonata-project/doctrine-orm-admin-bundle        3.4.2   Symfony Sonata / Integrate Doctrine ORM into the SonataAdminBundle
sonata-project/exporter                         1.8.0   Lightweight Exporter library
sonata-project/intl-bundle                      2.4.0   Symfony SonataIntlBundle
sparkling/vat-bundle                            0.2.0   A Symfony2 Bundle that checks a european VAT number
stfalcon/tinymce-bundle                         v2.2.0  This Bundle integrates TinyMCE WYSIWYG editor into a Symfony2 project.
stof/doctrine-extensions-bundle                 v1.3.0  Integration of the gedmo/doctrine-extensions with Symfony2
swiftmailer/swiftmailer                         v6.0.2  Swiftmailer, free feature-rich PHP mailer
sylius-labs/association-hydrator                v1.1.0  Doctrine ORM hydration performance optimization made easier.
sylius/sylius                                   v1.1.1  E-Commerce platform for PHP, based on Symfony framework.
symfony/assetic-bundle                          v2.8.2  Integrates Assetic into Symfony2
symfony/monolog-bundle                          v3.2.0  Symfony MonologBundle
symfony/phpunit-bridge                          v3.4.5  Symfony PHPUnit Bridge
symfony/polyfill-apcu                           v1.7.0  Symfony polyfill backporting apcu_* functions to lower PHP versions
symfony/polyfill-iconv                          v1.7.0  Symfony polyfill for the Iconv extension
symfony/polyfill-intl-icu                       v1.7.0  Symfony polyfill for intl's ICU-related data and classes
symfony/polyfill-mbstring                       v1.7.0  Symfony polyfill for the Mbstring extension
symfony/security-acl                            v3.0.1  Symfony Security Component - ACL (Access Control List)
symfony/swiftmailer-bundle                      v3.2.0  Symfony SwiftmailerBundle
symfony/symfony                                 v3.4.5  The Symfony PHP framework
symfony/thanks                                  v1.0.6  Give thanks (in the form of a GitHub ⭐) to your fellow PHP package maintainers (not limited to Symfony components)!
theseer/tokenizer                               1.1.0   A small library for converting tokenized PHP source code into XML and potentially other formats
twig/extensions                                 v1.5.1  Common additional features for Twig that do not directly belong in core
twig/twig                                       v2.4.6  Twig, the flexible, fast, and secure template language for PHP
webmozart/assert                                1.3.0   Assertions to validate method input/output with nice error messages.
white-october/pagerfanta-bundle                 v1.1.2  Bundle to use Pagerfanta with Symfony2
willdurand/hateoas                              2.10.0  A PHP library to support implementing representations for HATEOAS REST web services
willdurand/hateoas-bundle                       1.4.0   Integration of Hateoas into Symfony2.
willdurand/js-translation-bundle                2.6.6   A pretty nice way to expose your translation messages to your JavaScript.
willdurand/jsonp-callback-validator             v1.1.0  JSONP callback validator.
willdurand/negotiation                          v2.3.1  Content Negotiation tools for PHP provided as a standalone library.
winzou/state-machine                            0.3.3   A very lightweight yet powerful PHP state machine
winzou/state-machine-bundle                     v0.3.1  Bundle for the very lightweight yet powerful PHP state machine
zendframework/zend-code                         3.3.0   provides facilities to generate arbitrary code using an object oriented interface
zendframework/zend-eventmanager                 3.2.0   Trigger and listen to events within a PHP application
zendframework/zend-hydrator                     2.3.1  
zendframework/zend-stdlib                       3.1.0
```

**Comment by PapsOu** [comment]
Seems to be related to this closed issue, but the problems still occurs : https://github.com/symfony/symfony/issues/26364

**Comment by stof** [comment]
@PapsOu you are using 3.4.5, so an issue affecting 3.4.5 will of course impact you. The fix will be part of the 3.4.6 release (which should be released today due to this important bug).
for now, getting the patch requires using the dev version.

I'm closing this issue as duplicate.

**Comment by PapsOu** [comment]
Thanks @stof for these informations.
